test/classes/initializers/phase1/useAfterInit/passThisFn.good

```diff
@@ -0,0 +1,2 @@
+passThisFn.chpl:10: In initializer:
+passThisFn.chpl:15: error: cannot call a method during phase 1 of initialization
```

Comment:
Hmmm, fair point. I'll add a future for this case because I suspect we can't catch this reliably yet. For example, it's possible to define a method only on a particular instantiation of a type.

Reply:
You mean like

```chapel
record R {
  type t;
  proc foo() where t == int { writeln("Hello from int foo"); }
}
```

and this error is generated before the type is instantiated enough to know if the method resolves?

Reply:
Yes, something like the example you wrote is problematic for the current compiler architecture. Internally we have briefly discussed the likely need to move some error checking until after types are resolved, but that probably won't happen until after the 1.17 release.